clients/src/main/resources/common/message/RequestHeader.json

```diff
@@ -16,15 +16,18 @@
 {
   "type": "header",
   "name": "RequestHeader",
-  "validVersions": "0",
+  // Version 0 of the RequestHeader is only used by v0 of ControlledShutdownRequest.
+  //
+  // Version 1 is the first version with ClientId.
+  "validVersions": "0-1",
```

Comment:
Assume that the previous schema was incorrect and we always had version 1.

Reply:
Hmm, I'm not sure I follow.  Schema version 0 isn't incorrect if it is sent for v0 of ControlledShutdownRequest.  Of course, if we were doing it all over again, we wouldn't have this irregularity.  But we have to handle it now, for compatibility reasons.